SU2_CFD/src/solvers/CSpeciesFlameletSolver.cpp

```diff
@@ -427,7 +427,7 @@ void CSpeciesFlameletSolver::BC_Isothermal_Wall_Generic(CGeometry* geometry, CSo
   for (unsigned long iVertex = 0; iVertex < geometry->nVertex[val_marker]; iVertex++) {
     unsigned long iPoint = geometry->vertex[val_marker][iVertex]->GetNode();
 
-    if (cht_mode) temp_wall = GetConjugateHeatVariable(val_marker, iVertex, 0);
+    if (cht_mode) temp_wall = solver_container[FLOW_SOL]->GetConjugateHeatVariable(val_marker, iVertex, 0);
```

Comment:
In this case does the flamelet solver need conjugate heat variables at all?
Also I don't think this would work for weakly coupled energy, where the temperature is stored in the heat solver instead of the flow solver.

Reply:
The flamelet solver indeed doesn't need conjugate heat variables anymore. I just removed those lines. Regarding the weakly coupled energy, I don't know for sure. When running flamelet simulations, the energy transport equation is solved by the flamelet solver, not the flow solver, so my best guess is that weakly coupled heat should be incompatible with flamelet simulations.

Reply:
I discussed with @bigfooted regarding the compatibility of the flamelet solver with the weakly coupled heat solver. We concluded that it does not make much sense to use both weakly coupled heat and the flamelet fluid model in the same zone, as the flamelet fluid model computes the temperature based on the transport of controlling variables. This prevents the temperature from being an independent, transported scalar variable, as is the case for the weakly coupled heat option.